protoutils/protoutils_test.go

```diff
@@ -370,7 +379,7 @@ type (
 )
 
 type OmitStruct struct {
-	X float64 `json:"x"`
+	X float64 `json:"x,omitempty"`
```

Comment:
Ah I missed this before - you can revert and I see that there's a test for `{"omit struct", OmitStruct{}, map[string]interface{}{"x": 0.0}`, thanks!